project/PATCH-RELEASES.md

```diff
@@ -42,6 +42,9 @@ review sessions take place on a weekly basis, more frequent if needed:
 - P1, P2, and P3 issues/PR's should not influence the decision, and
   should be moved to the X.Y.Z+1 milestone, or removed from the
```

Comment:
That's right, formulated this way it means that we never ship P2 and P3 as part of a patch release... I need to rework this.